src/notification-banner/private/_NotificationBannerView.jsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
```

Comment:
This is a port of the existing terra-alert implementation with minor changes to its responsive size handling and accessibility labels.